### PR TITLE
Add DisplayName content

### PR DESCRIPTION
**Pull request by Rythian2277**
Adds:
- Display Name box to `!whois` if the user has a set DisplayName which is different to their Username,
- %DISPLAYNAME% format option => Shows the DisplayName if valid, Username if none set,
- %SMARTNAME% format option (as per eryn's suggestion on the matter) => Shows `DISPLAYNAME {USERNAME}` if the user has a DisplayName, `USERNAME` if no DisplayName exists.

***Before review/merge, I would like others to test this code as DisplayNames are still in the process of being rolled out to the community.***

### CODE REVIEW

**Comment by Rythian2277** [comment]
For some reason this closed, probably due to tidying of my forked repo. Still a WIP but the feature will be available upon test completion

**Comment by Rythian2277** [comment]
As far as I am aware, this is operational and has been running successfully on my forked version of RoVer since this PR was made, I would like someone else just to do a final review of the code to ensure I haven't messed anything up though.

File: src/commands/rover/WhoisCommand.js [hunk 1]
```diff
@@ -179,6 +179,14 @@ class WhoisCommand extends Command {
           ]
         }
 
+        if (data.robloxUsername !== apiUserData.displayname) {
```
Comment (Wolftallemo) [inline src/commands/rover/WhoisCommand.js:182]:
```suggestion
        if (data.robloxUsername !== apiUserData.displayName) {
```

File: src/commands/rover/WhoisCommand.js [hunk 2]
```diff
@@ -179,6 +179,14 @@ class WhoisCommand extends Command {
...
+          embed.fields.push({
+            name: "Display Name",
+            value: apiUserData.displayname,
```
Comment (Wolftallemo) [inline src/commands/rover/WhoisCommand.js:185]:
```suggestion
            value: apiUserData.displayName,
```

**Review by Wolftallemo — APPROVED** [review]
After some fixes, it appears to work correctly. I'll leave it for now for others to test.

**Comment by Wolftallemo** [comment]
Ry's test bot hasn't seen any reported issues, so I'm merging.

**Pull request merged**